app/provider/app.go

```diff
@@ -535,6 +504,9 @@ func New(
 
```

Comment:
Out of scope, but on new line 502, we keep using the ModuleName instead of the port id const (which doesn't exist in the provider). Curious to know why it's different here

Reply:
For provider, the port ID is `provider` and the module name is `provider`. The change in routing logic that broke consumers happened because the module name is `ccvconsumer` and the port is `consumer`. So I had to change it in consumers but not in providers.